File: modul-03/bahasa c/PRAK304-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

int main() {
    int nilai;
    char *hasil = NULL;

    scanf("%d", &nilai);

    if (nilai >= 100) {
        hasil = "Anda Menginput Melebihi Limit Bilangan";
    } else if (nilai <= 100 && nilai >= 20) {
        hasil = "Puluhan";
    } else if (nilai < 20 && nilai >= 10) {
        hasil = "Belasan";
    } else if (nilai <= 9 && nilai >= 1) {
        hasil = "Satuan";
    } else if (nilai == 0) {
        hasil = "Nol";
    } else {
        printf("Tidak terdefinisi\n");
        return 0;
    }

    printf("%d = %s\n", nilai, hasil);

    return 0;
}

File: modul-03/python/PRAK304-2310817310003-Mfaridsetiawan.py
nilai = int(input(' '))

hasil = None

if nilai >= 100:
   hasil = 'Anda Menginput Melebihi Limit Bilangan'

elif nilai <=100 and nilai >= 20:
     hasil = 'Puluhan'

elif nilai < 20 and nilai >=10:
     hasil = 'Belasan'

elif nilai <= 9 and nilai >= 1:
     hasil = 'Satuan'

elif nilai == 0:
     hasil = 'Nol'

print(' {} = {}'.format(nilai, hasil))